index.html

```diff
@@ -366,8 +386,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_LINK</code> and object attribute
 											<code>xml-roles:doc-backlink</code>. </p></td>
-								<td>AXRole: <code>AXLink</code><br /> AXSubrole: <code>&lt;nil&gt;</code><br />
-									AXRoleDescription: <code>'link'</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXLink</code></li>
+										<li>AXSubrole: <code>&lt;nil&gt;</code></li>
+										<li>AXRoleDescription: <code>'link'</code></li>
```

Comment:
TBD... Should this be a new subrole and should it include "back" in the role description.

Reply:
Leaving as-is. Can revisit later if needed.

Reply:
James couldn't browsers also check to see if the role is doc-backlink and apply different CSS styling to denote a backlink vs. a regular link? Not sure this would only be accessed by AT only.
I personally would like to see "backlink" in an additional 
<li>AXCustomContent:<code>{ label: "type", value: "backlink" }</code></li>
or
<li>AXRoleDescription: <code>'backlink'</code></li>

Reply:
Is there a risk of AT detection here beyond what would exist for any link? Back links aren't only for AT users. ebook readers lack back buttons, so they're widely available to everyone in publications. It's used as a means of getting back to a footnote reference, for example.

I agree the context of getting the user back to the starting reference is helpful here, so having "back" in the description would be useful.

Reply:
> Is there a risk of AT detection here beyond what would exist for any link? 

Yes. Any activation that does not include a prior focus, or other pointer-related events, etc. would indicate the potential for AT detection. Some of the factors could be mitigated, but without further guidance from the TAG on https://github.com/w3ctag/design-principles/issues/293 I would rather address that separately in a standalone issue.